requirements/base.txt

```diff
@@ -10,4 +10,5 @@ tqdm~=4.66
 typing_extensions~=4.7.1
 owlready2
 networkx
-pydantic==2.5.3
\ No newline at end of file
+pydantic==2.5.3;python_version<="3.7"
```

Comment:
Wow, didn't know you could do that, that's pretty cool to know!

Reply:
Right!?